sdks/python/apache_beam/examples/dataframe/README.md

```diff
@@ -0,0 +1,142 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Example DataFrame API Pipelines
+
+This module contains example pipelines that use the [Beam DataFrame
+API](https://beam.apache.org/documentation/dsls/dataframes/overview/).
+
+## Pre-requisites
+
+You must have `apache-beam>=2.30.0` installed in order to run these pipelines,
+because the `apache_beam.examples.dataframe` module was added in that release.
+Additionally using the DataFrame API requires `pandas>=1.0.0` to be installed
+in your local Python session. The _same_ version should be installed on workers
```

Comment:
You might link to https://github.com/apache/beam/blob/release-2.30.0/sdks/python/container/base_image_requirements.txt#L61 for the installed version in beam by default.

Reply:
Done